File: test_playstyles.mjs
import fs from 'fs';

async function run() {
  const headers = JSON.parse(fs.readFileSync('./headers.json', 'utf8'));
  const payload = {
    query: { bool: { must: [{ query_string: { fields: ['assetId'], query: '3114943' } }] } },
    from: 0,
    size: 1
  };
  const response = await fetch('https://renderz.app/api/search/elasticsearch', {
    method: 'POST',
    headers: headers,
    body: JSON.stringify(payload)
  });
  const data = await response.json();
  console.log(JSON.stringify(data, null, 2));
}
run().catch(console.error);

File: test_playstyles.py
import json
import requests

with open("headers.json", "r") as f:
    headers = json.load(f)

payload = {
    "query": {"bool": {"must": [{"query_string": {"fields": ["assetId"], "query": "3114943"}}]}},
    "from": 0, "size": 1
}
resp = requests.post("https://renderz.app/api/search/elasticsearch", json=payload, headers=headers)
data = resp.json()
print(json.dumps(data, indent=2))
